src/components/navbar.js

```diff
@@ -1,39 +1,53 @@
-import React, {useState ,useEffect } from 'react'
+import React, { useState, useEffect } from 'react'
 import '../App.css'
 import '../pages/buy'
-const Navbar =()=> {
+import { Link } from 'react-router-dom'
+// import { UseContext } from '../App'
+// import { useContext } from 'react'
+// import { Count } from '../App'
+import { useContext } from 'react'
+import { CountContext } from '../App'
+// import { counter } from '@fortawesome/fontawesome-svg-core'
+
+const Navbar = ({ cartCount }) => {
+    const countContext = useContext(CountContext)
+    console.log(countContext.countDispatch)
+    // const usercount = useContext(Count)
+    // console.log(usercount)
     const [navbar, setNavbar] = useState(false)
     const Sticknavbar = () => {
         if (window.scrollY >= 50) {
             setNavbar(true)
             console.log(window.scrollY)
-    } else {
-      setNavbar(false)
+            // console.log(cartCount)
+        } else {
+            setNavbar(false)
+        }
     }
-  }
-    useEffect(()=>{
+    useEffect(() => {
         Sticknavbar()
         window.addEventListener('scroll', Sticknavbar)
-    },[])
+    })
+    console.log(countContext.countState.cart.length)
     return (
-        <nav className= {navbar? "nav-container sticky ": "nav-container"}  id='nav'>
+        <nav className={navbar ? "nav-container sticky " : "nav-container"} id='nav'>
             <div className="left-nav">
                 <img src="" alt="" />
                 {
-                navbar?<div className="search" >
-                <input type="text" placeholder='Search Product'/>
-                <button className="primary-btn"><i className="fa-search"></i></button></div>: <></> 
-             }
+                    navbar ? <div className="search" >
```

Comment:
this can be done like : navbar && (content)
that would give the same result but it is a cleaner approach